### DISCUSSION TITLE
GR-Procedure-PlaceOfPerformance (change of the nodeId attribute in SDK 1.12)

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#1002] [Q&A] [answered]
The NodeId attribute of this group has been changed in the noticesTypes files.

Should I change the Node Id attribute to the value “ND-Procedure PlacePerformanceAdditionalInformation”. This is the value that this group had in previous SDKs?

![image](https://github.com/user-attachments/assets/99f1c4a0-0a31-4a36-b260-dc834bb66985)

Thanks in advance,
Alex

**Comment by YvesJo** [comment, marked as answer]
Hi,
Be aware that nodes may need redefinition from one SDK to the other.
Association of Groups and nodes is defined per SDK version, so if your intention is to use a specific SDK, you should make sure the correct associations are in place.
KR
